packages/extension/src/background.ts

```diff
@@ -169,6 +214,35 @@ browser.runtime.onConnect.addListener(function (port) {
       case "RESET_ALL": {
         return browser.storage.local.clear()
       }
+
+      case "ADD_WHITELIST": {
+        return addPendingWhitelist(data)
+      }
```

Comment:
Braces shoudn't be needed around `case`s??
```suggestion
      case "ADD_WHITELIST":
        return addPendingWhitelist(data)
```

Reply:
that's right, but I prefer them. 
JS and TS are curly brace languages and I prefer to use them over indentations, which is an anti pattern in JS.
it also helps to not forget return, as it'd otherwise execute multiple cases :)

Reply:
It's fine, it's just that I had never seen this pattern in any other curly brace language project in the past, so I was a bit surprised as to the reason. I thought it was a workaround for the lack of semicolons

Reply:
Also there might be an eslint rule that can help you not forget return statements in switch cases